### ISSUE TITLE
how did you add capital like example image 1

### ISSUE THREAD

**Issue by denemier** [Darakah/obsidian-map#12]

**Comment by Darakah** [comment]
Are you on the obsidian discord? Better to send question as DM on discord then posting Issues on github if possible. You can find me as `darakah` on the obsidian official discord. 

Are you talking about the category `Capital`? If yes, then you add it through the settings tab of the plugin.
Are you talking about the pop-up on the right? Clicking on the marker or overlay that is linked to an obsidian note will render the note in a pop-up as the one shown in the image.

**Comment by denemier** [comment]
no i am not on the discord yet,
but i found the settings as you said and have added custom marker and can see it also in markers option (bottom left)
but i dont understand adding marker that is linked to an obsidian note:

i added this to a note like release notes, but doesnt work on map is this not how it works?

Added note Geo/Temporal block info:
`
<span class="ob-world-map" 
	  data-type="Capital"
	  data-loc="9.380052 / 9.969492"
	  data-dateStart="20"
	  data-dateEnd="90"
	  data-icon="map-marked-alt"
	  data-color="red"
	  data-marker='<img class="map-custom-marker-img" 
		       src="app://local/Users/Resources/Icon/Layer_Potion_Shop.png"/>'/>
`

**Comment by denemier** [comment]
i changed image to external web image link and deleted data marker option still doesnt show on the map btw i use openstreetmap as you said

**Comment by Darakah** [comment]
What was the name of the category you added?

the `data-type` specifies the group in the map control that it will be bound to

You need to search for the markers to be added. On the top right, click the gears, input any year that is between your start and end date and click filter. Then it will show up on the map

**Comment by denemier** [comment]
![image](https://user-images.githubusercontent.com/63566495/115962626-03d28c80-a4b8-11eb-86fe-e67f472f9cf9.png)
i have written your release note wxample to my example.md note changed image, and also tried without backslashes before equal sign what i am missing?

**Comment by denemier** [comment]
right now trying with Original Capital marker

**Comment by Darakah** [comment]
1. Copy paste this section into an md note
```html
<span class="ob-world-map" 
	  data-type="Capital"
	  data-loc="9.380052 / 9.969492"
	  data-dateStart="20"
	  data-dateEnd="90"
	  data-icon="map-marked-alt"
	  data-color="red"/>
```

2. Make sure the note is saved, reload obsidian so that the map info updates
3. Click the map icon (globe icon) in the left ribbon
4. Click the gears button on the top right 
5. Write '70' where it says `year`
6. Click `Filter` button in the same tab
7. Check the `Capital` box in the bottom left
8. Look at the map, an icon should be added
9. Click icon

**Comment by denemier** [comment]
an icon is added now but  no matter hwo many notes i have added your copy paste section, there is always 1 icon on the map, and what interesting is, its somehow showing a specific galleryinfo file from your gallery plugin

**Comment by Darakah** [comment]
Did you change the `data-loc` for all those copy paste notes? Otherwise they would show up exactly on top of each other...

**Comment by denemier** [comment]
yes i changed it a little bit like 9 to 29, i am just deleting the plugin and fresh install and do again what you said

**Comment by Darakah** [comment]
You also need to restart obsidian so that the map info is update. It is rebuilt on start-up

**Comment by denemier** [comment]
new vault, new plugin install no result as in video

https://user-images.githubusercontent.com/63566495/115964371-e5708f00-a4bf-11eb-830b-090c590ba354.mp4

and the plugin folder includes only 3 files and 1 later generated file from the release like seen in this screenshot below
![image](https://user-images.githubusercontent.com/63566495/115964418-27013a00-a4c0-11eb-9439-c071462ba7bc.png)

**Comment by Darakah** [comment]
1. You need to write a year, the scroller doesn't work (same for the dropdown, these are just placeholder elements for something that I want to do in the future). As I mention above, you need to TYPE, `70` in the box where it says YEAR (right above where you click the stuff). Then click filter. 
2. Something is weird with your span block, it should be HIDDEN, i.e. doesn't show in the preview mode

**Comment by denemier** [comment]
still not working, probably not working correctly related to your second list item

**Comment by denemier** [comment]
hey again, i found a weird way and it works:
- so a markdown note file still doesn't work as said before
- but because also said the span block should be hidden in preview mode, i tried this time adding span block inside a images info-block from your gallery plugin, and when previewing the info file it's not showing the span block as excepted and also after closing and opening the program i can also see all images related to locations, this is very good
- but isn't this plugin originally made for the note files?
